tests/components/temper/test_sensor.py

```diff
@@ -0,0 +1,34 @@
+"""The tests for the temper (USB temperature sensor) component."""
+from datetime import timedelta
+from unittest.mock import Mock, patch
+
+from homeassistant.setup import async_setup_component
+import homeassistant.util.dt as dt_util
```

Comment:
```suggestion
from homeassistant.setup import async_setup_component
from homeassistant.core import HomeAssistant
import homeassistant.util.dt as dt_util
```

Import needed for type hints.